WcaOnRails/config/routes.rb

```diff
@@ -36,6 +36,7 @@
     get 'results/all' => 'competitions#show_all_results'
     get 'results/by_person' => 'competitions#show_results_by_person'
 
+    get 'registrations/event_fee_for_selected_events' => 'registrations#entry_fee_for_selected_events', as: :entry_fee_for_selected_events
```

Comment:
nit: We prefer to use dashes in urls. I know there are a few places where we used underscores, but we're trying to avoid adding any more of them at least!